routes/jobs.js

```diff
@@ -6,7 +6,19 @@ const progression = require('../lib/progression');
 const moment = require('moment');
 
 function parseDeadline(deadlineString) {
-  return deadlineString ? moment(deadlineString, 'DD/MM/YYYY').format() : null;
+  const dateFormat = !deadlineString.includes('-') ? 'DD/MM/YYYY' : undefined;
+  return deadlineString ? moment(deadlineString, dateFormat).format() : null;
+}
+
+function validateDeadline(deadline, req) {
+  if (deadline) {
+    if (deadline.includes('-')) {
+      req.checkBody('deadline', 'Deadline should be in "dd/mm/yyyy" format').isDate();
```

Comment:
This one is tricky. It is correct browser -> on browser we want to push user to use 'DD/MM/YYYY'.
On mobile there will be native date support so this error should not be shown in theory.